test/initials-test.js

```diff
@@ -119,25 +141,26 @@ test('initials.parse(nameOrNames, {existing: initialsForNames})', function (t) {
         'John Doe': 'JD'
       }
     }), [{name: 'John Doe', initials: 'JD'}, {name: 'Jane Dane', initials: 'JDa'}], 'respect existing initials  for multiple names')
-  t.end()
+
 })
 
 test('initials(), no params', function (t) {
+  t.plan(4)
+
   t.equal(initials(), '', 'initials() without nameOrNames, no initials')
   t.equal(initials.addTo(), '', 'initials.addTo() without nameOrNames, no initials')
   t.deepEqual(initials.parse(), {}, 'initials.parse() without nameOrNames, no initials')
 
   t.deepEqual(initials(['', '']), ['', ''], 'initials with multiple persons but no names')
-  t.end()
 })
 
 test('initials(), name.length is less 3', function (t) {
+  t.plan(2)
+
   t.equal(initials('K'), 'K', 'name.length is < 2, so the initials are equal to name')
   t.equal(initials('Mo'), 'Mo', 'name.length is < 3, so the initials are equal to name')
-  t.end()
 })
 
-// uncomment the block below to run a single test only
-// test.only('debug', function (t) {
-//   t.equal(initials('foo'), 'bar', 'debugging')
-// })
+// describe('debug', function() {
+//   equal( initials('John Doe', 3), 'JDo', 'John Doe ☛ JDo' );
+// });
```

Comment:
Can you leave the block above with the `test.only`, instead of the current `describe` one, as this one is obsolete?